vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/private_cdn/get_publisher/get_publisher.cc

```diff
@@ -0,0 +1,268 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "bat/ledger/internal/endpoint/private_cdn/get_publisher/get_publisher.h"
```

Comment:
Nit: Add a blank line here. Seeing this in most of the .cc files, not sure if it's intentional so I won't flag them all.